src/utils/helper.js

```diff
@@ -1,11 +1,33 @@
 import fs from 'fs';
 import path from 'path';
+import toml from 'toml';
+
+function readAndParseTomlConfig(filePath) {
+    try {
+        const content = fs.readFileSync(filePath, 'utf-8');
+        return toml.parse(content);
+    } catch (err) {
+        console.error('Error reading or parsing TOML file:', err);
+        process.exit(-1);
+    }
+}
+
+//Checking if the directory exists and if not, create it
+function checkAndCreateDirectory(dirPath) {
+  if (!fs.existsSync(dirPath)) {
+      fs.mkdirSync(dirPath, { recursive: true });
+  }
+}
 
 function sanitizeInputCommand(args) {
   let pathToInputFileOrDir = args[0];
   let outputCommand = args[1];
   let pathToOutputDir = args[2];
 
+  // Check if input file or directory exists and if not, create it
+  checkAndCreateDirectory(pathToInputFileOrDir);
+  checkAndCreateDirectory(pathToOutputDir);
```

Comment:
Is there any reason why we're checking and creating a directory here? The existing code in this function takes care of this already, to check and create an output file or dir if it doesn't exist. Also as far as input file is concerned, we shouldn't be creating an input file or an input directory, if it doesnt exist then let line 32 take care of it and exit the execution of the program.

Reply:
Solved in commit [d21ede3a](https://github.com/sdthaker/Node-TILify/pull/13/commits/d21ede3afba2b24850188d116d612d4b4c13062d)